peer_review_data/main.py

```diff
@@ -12,50 +12,47 @@
 LOGGER: Logger = getLogger(__name__)
 
 
-def saveCourseAndUsers(canvasCourse: CanvasCourse) -> (bool, Course):
+def saveCourseAndUsers(canvasCourse: CanvasCourse) -> Course:
     course = models.Course.fromCanvasCourse(canvasCourse)
     LOGGER.info(f'Saving {course}…')
     course.save()
 
     LOGGER.info(f'Saving users of {course}…')
-    for canvasUser in canvasCourse.get_users():
+    '''
+    Possible bug in `canvasapi` module…
+    When calling `get_users` with the argument `include='test_student'`, the
+    module adds it to the URL as `include=test_studentanalytics_url`.  The
+    origin of the "analytics_url" string is unknown.  However, it's known that
+    the Canvas API expects the "include" argument to appear as `include[]`, so
+    by using a dictionary here with the key in the correct format, the bug is
+    bypassed and the argument appears correctly.
+    Strangely, a later call to `get_rubric` uses the `include` keyword argument
+    without problem.
+    '''
+    for canvasUser in canvasCourse.get_users(**{'include[]': 'test_student'}):
```

Comment:
Why are we including the test student?

Reply:
Some assessments were made by the test student.  If the test student's ID wasn't included in the user table, then saving its assessments would cause an error about the missing user.

Reply:
Opened #12.